keps/sig-node/2570-memory-qos/README.md

```diff
@@ -87,22 +89,171 @@ Cgroups v2 introduces a better way to protect and guarantee memory quality.
 
-We also introduce `memory.high` to throttle container memory overcommit allocation. It will be set based on a formula:
+We also introduce `memory.high` for container cgroup to throttle container memory overcommit allocation. 
+***Note***: memory.high is set for container-level cgroup, and not for pod-level cgroup. If a container in a pod sees a spike in memory usage, it could result in total pod-level memory usage to reach memory.high level set at pod-level cgroup. This will induce throttling in other containers as the pod-level memory.high was hit. Hence to avoid containers from affecting each other, we set memory.high for only container-level cgroup.
+
+#### Alpha v1.22
+It is based on a formula:
 ```
-memory.high=limits.memory/node allocatable memory * memory throttling factor
+memory.high=(limits.memory or node allocatable memory) * memory throttling factor, 
+where default value of memory throttling factor is set to 0.8
 ```
 
-e.g. if a container has `requests.memory=50, limits.memory=100`, and we have a throttling factor of .8, `memory.high` would be 80. if a container had no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .8 to that value.
-It must be ensure that `memory.high` is always greater than `memory.min`.
+e.g. If a container has `requests.memory=50, limits.memory=100`, and we have a throttling factor of .8, `memory.high` would be 80. If a container has no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .8 to that value.
+It must be ensured that `memory.high` is always greater than `memory.min`.
 
 Node reserved resources(kube-reserved/system-reserved) are either considered. It is tied to `--enforce-node-allocatable` and `memory.min` will be set properly.
 
 Brief map as follows:
 | type | memory.min | memory.high |
 | -------- | -------- | -------- | 
 | container | requests.memory | limits.memory/node allocatable memory * memory throttling factor |
-| pod | sum(requests.memory) | n |
-| node | n/a | pods, kube-reserved, system-reserved | n |
+| pod | sum(requests.memory) | N/A |
+| node | pods, kube-reserved, system-reserved | N/A |
+
+#### Alpha v1.27
+The formula for memory.high for container cgroup is modified in Alpha stage of the feature in K8s v1.27. It will be set based on formula:
+```
+memory.high=floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize, where default value of memory throttling factor is set to 0.9
+```
+Note: If a container has no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .9 to that value.
+
+
+The table below runs over the examples with different values requests.memory and 1Mi pageSize:
```

Comment:
nit: In real world it would not be 1Mi page size, since page size default at least on x86 is 4KB, so would need to round on nearest page size boundry when coverting to bytes. But no need to change anything here, since we want to show table in Mi units.